docs/parcels.md

```diff
@@ -0,0 +1,110 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
```

Comment:
Grammar nitpick: this is a run on sentence that I think we could split into multiple sentences (each comma separated part is its own independent clause and could be a sentence in its own right)

Reply:
What does "imperatively" mean? I know that you are referring to something that is mounted with a function call instead of based on routing, but I don't think the lay person would, at first.